backend/graphql/resolvers.ts

```diff
@@ -0,0 +1,166 @@
+import { getGenres, getTracks, createTrack, updateTrack, getTrackById, deleteTrack, deleteMultipleTracks, saveAudioFile, deleteAudioFile } from '../utils/db';
+import { getAllTracks, getTrack, addTrack, updateTrackById, removeTrack, removeTracks } from '../controllers/tracks.controller';
+import { createSlug } from '../utils/slug';
+
+export const resolvers = {
+  Query: {
+    health: async () => 'ok',
+    genres: async () => {
+      return getGenres();
+    },
+    tracks: async (_parent: any, args: any) => {
+      const params = {
+        page: args.page,
+        limit: args.limit,
+        sort: args.sort,
+        order: args.order,
+        search: args.search,
+        genre: args.genre,
+        artist: args.artist,
+      };
+      const { tracks, total } = await getTracks(params);
+      const page = params.page || 1;
+      const limit = params.limit || 10;
+      return {
+        data: tracks,
+        meta: {
+          total,
+          page,
+          limit,
+          totalPages: Math.ceil(total / limit)
```

Comment:
Nitpick: For page and limit you make operations on separate lines 22-23, but here it is inlined in meta object.
I would suggest making it consistent and make all three const like page and limit or none of them.